package.json

```diff
@@ -11,9 +11,9 @@
     "snyk": "snyk test",
     "clean": "rimraf ./dist && mkdir -p ./dist ./dist/js ./dist/css ./dist/fonts ./dist/img",
     "start": "./server/server.js",
-    "js": "browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js > ./dist/js/app.min.js",
+    "js": "browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -t [ envify --HMDA_API https://192.168.99.100:4443/hmda ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js > ./dist/js/app.min.js && browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map silent_renew.js.map --output ./dist/js/silent_renew.js.map --uglify [ -cm ] ] ./src/js/silent_renew.js > ./dist/js/silent_renew.js",
     "sass": "node-sass --quiet --output-style=compressed --source-map-embed --source-map=./dist/css/ ./src/sass/app.scss ./dist/css/app.min.css && cp node_modules/uswds/src/fonts/sourcesanspro-* ./dist/fonts",
-    "html": "html-minifier --collapse-whitespace ./src/index.html -o ./dist/index.html",
+    "html": "html-minifier --collapse-whitespace ./src/index.html -o ./dist/index.html, && html-minifier --collapse-whitespace ./src/silent_renew.html -o ./dist/silent_renew.html",
```

Comment:
Minor but I think you can remove the `,`  before the `&&` from `./dist/index.html, &&`.